storm-server/src/test/java/org/apache/storm/scheduler/resource/strategies/scheduling/TestDefaultResourceAwareStrategy.java

```diff
@@ -491,7 +653,10 @@ public void testDefaultResourceAwareStrategy() {
     /**
      * test if the scheduling logic for the DefaultResourceAwareStrategy (when made by network proximity needs.) is correct
      */
-    public void testDefaultResourceAwareStrategyInFavorOfShuffle() {
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    public void testDefaultResourceAwareStrategyInFavorOfShuffle(boolean useDeprecatedConfigForProximity)
```

Comment:
Should we remove this? I don't think we have `DeprecatedConfigForProximity` in community storm